### PR TITLE
Update .gitignore file #41

### PR DESCRIPTION
**Pull request by techytushar**
PR for issue #41 
Signed-off-by: Tushar Mittal <chiragmittal.mittal@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+/Include
+/include
+/pip-selfcheck.json
+
+# IDEs
+.project
+.pydevproject
+.idea
+org.eclipse.core.resources.prefs
+.vscode
```
Comment (yash-nisar) [inline .gitignore]: A new line at the end of file would be nice :).

**Comment by yash-nisar** [comment]
Did you investigate why circleci fails ?

File: .gitignore (hunk 1 above)
Comment (yash-nisar) [reply .gitignore]: Can you add `.pytest_cache` too under the IDEs section ?

**Comment by techytushar** [comment]
`brew install pyenv` gives error ` Homebrew must be run under Ruby 2.3! You're running 2.0.0. (RuntimeError)` any suggestions @yash-nisar

**Comment by yash-nisar** [comment]
Can you trigger a rebuild because that error has been resolved ?

**Comment by pombredanne** [comment]
This looks fine to me. merging! Thank you!

**Comment by yash-nisar** [comment]
@pombredanne @krysnuvadga Can you please merge this one since it has already been approved ?

**Pull request merged**